File: library/src/main/java/com/getstream/sdk/chat/rest/response/EventResponse.java
package com.getstream.sdk.chat.rest.response;

import com.getstream.sdk.chat.model.Event;
import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class EventResponse {
    @SerializedName("event")
    @Expose
    private Event event;

    @SerializedName("duration")
    @Expose
    private String duration;

    public Event getEvent() {
        return event;
    }
}

File: stream-chat-android-client/src/main/java/io/getstream/chat/android/client/api2/model/response/EventResponse.kt
package io.getstream.chat.android.client.api2.model.response

import com.squareup.moshi.JsonClass
import io.getstream.chat.android.client.api2.model.dto.ChatEventDto

@JsonClass(generateAdapter = true)
internal data class EventResponse(
    val event: ChatEventDto,
    val duration: String,
)
